docs/how-to/ocm-control-plane-walkthrough.md

```diff
@@ -381,6 +381,22 @@ So now we have a working gateway with DNS and TLS configured. Let place this gat
 ## Watching DNS changes
 If you want you can use ```watch dig $MGC_SUB_DOMAIN``` to see the DNS switching between the two addresses 
 
+## Clean up local environment
+> **Note:** :thought_balloon: The `quickstart-setup.sh` cleans up automatically before the run
+
+In any terminal window target control plane cluster by:
```

Comment:
I'm wondering if we need to add this here. Reason being that with the recent quickstart change this wallkthrough doesn't required a clone of the repo. But introducing some make commands does.

Reply:
fair point. Removed